css/obBase.css

```diff
@@ -244,7 +244,7 @@ form {
   background-color: #252525;
   border-left: 0;
   height: calc(100% - 50px);
-  z-index: 2;
+  z-index: 999;
```

Comment:
Use the minimum z-index necessary, otherwise we get a z-index arms race with z-index 100000000000000000000 eventually.